pkg/controller/controller.go

```diff
@@ -885,3 +885,34 @@ func (c *Controller) GetMetrics(context context.Context, opt *pb.GetMetricsOpts)
 
 	return pb.GenericResponseResult(result), err
 }
+
+func (c *Controller) CollectMetrics(context context.Context, opt *pb.CollectMetricsOpts) (*pb.GenericResponse, error) {
+	log.Info("in controller collect metrics methods")
```

Comment:
This is a debug msg.  Should use level 5: log.V(5).Info

Reply:
updated